doc/nrf/protocols/bt/bt_qualification/index.rst

```diff
@@ -0,0 +1,124 @@
+
+A full list of Bluetooth product types including their descriptions is available on the `Bluetooth product types`_ page.
+
+
+Qualification process
+*********************
+
+The Bluetooth End Product qualification cannot be inherited.
+Hence, you must complete this process by yourself.
+You can either perform a full qualification process for a new product or edit an existing entry for a product that has already been qualified.
+
+The only way to qualify your product is to complete the `Bluetooth SIG's Qualification Process`_ under your member company's account.
+To check if your company is already a SIG member, go to `Join Bluetooth SIG`_ page and search the Member Directory list.
+
+Before you start the qualification process, review `PRD and DPD`_ documents, that will help you to understand the basics.
+
+Creating new entry
+==================
+
+Log into `Launch Studio`_, the official qualification tool, that will guide you through one of the following processes depending on the state of your product:
+
+* `Qualification process with no required testing`_ - If your complete product uses only pre-qualified End Products or Subsystems and implements APIs as instructed in the corresponding |NCS| documentation.
+* `Qualification process with required testing`_ - In the following cases:
+
+  * If your product uses a new Bluetooth design that is not yet qualified by Nordic Semiconductor.
+  * If your product uses a pre-qualified Component.
+  * If your product alters a previously qualified Bluetooth design by changing the code or configuring it in a way that can break the qualification requirements for the Nordic Semiconductor's qualified design.
+
+Updating existing entry
+=======================
+
+.. note::
+
+   An existing entry can be edited only by the person that completed the listing.
+   If you want to change the entry owner, contact SIG support.
+
+
+Update an existing qualification entry through `Launch Studio`_ in the following cases:
+
+* You changed an existing product's design.
+* Your new product can be added as a new entry to existing list of product entries as there was no design change in the new product.
+* You want to switch to another major |NCS| version for an existing product.
+* In some cases, if you want to switch to another minor |NCS| version for an existing product.
+
+It may happen that you would need to alter the configuration of the qualified design or slightly modify small parts of the existing design to better fit your application needs.
+If you make such changes, you must check the following:
+
+* Ensure that the resulting design does not add new features (ICS entries) that are not present in Nordic Semiconductor's qualified design.
+* Ensure that the resulting design does not break the qualification of the existing design.
+
+You can do so by running relevant qualification tests using test tools.
```

Comment:
You mean the PTS tool? Why the plural?

Reply:
This is explained in PRD and should be well known jargon for Bluetooth product makers.

Reply:
```suggestion
You can do so by running relevant qualification tests using test tools listed in the PRD.
```

Reply:
Tools are not listed in the PRD. "Qualification tests using test tools" is a well known concept. Those who are following a PRD would know it by virtue of lot of literature and material around this process.